ts_src/token.ts

```diff
@@ -0,0 +1,161 @@
+import * as tapyrus from 'tapyrusjs-lib';
+import { Utxo } from './utxo';
+import Wallet from './wallet';
+
+export interface Token {
+  transfer(
```

Comment:
Wouldn't it be better for wallet to implement the `transfer`?

Reply:
Updated at e1e3509